specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/Ocr.json

```diff
@@ -110,6 +110,85 @@
           }
         }
       }
+    },
+    "/read/core/asyncBatchAnalyze": {
```

Comment:
It looks like operation is a long-running operation. There's an Azure standard for how long-running operations should look/behave. I recommend looking at [Redis's create cache](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/redis/resource-manager/Microsoft.Cache/stable/2018-03-01/redis.json#L146-L202) operation for an example of how your operation can look.
Our generated SDKs can produce a much better interface for end-users if you follow the standard LRO structure.

Reply:
You can also go [here](https://github.com/shahabhijeet/adx-documentation-pr/blob/docLro/sdks/LRO/LRO_AzureSDK.md) for a description of the LRO standard.

Reply:
Thanks! It's indeed a long running operation and I have set `x-ms-long-running-operation` to `true`.

Reply:
Sorry. I have a question here. Our API doesn't implement in the way which is described in the LRO standard. We will return the async operation URL in the header of `Operation-Location`. However, from the LRO standard, we should put the URL in one of `Azure-AsyncOperation`, `Location` or `Retry-After`. So I'm afraid that `x-ms-long-running-operation` is not applicable here.

Reply:
You're correct. Your currently implementation and API specification does not adhere to the LRO standard, so I was recommending that you change your implementation and API specification so that it does adhere to the standard. It may delay you by a day or two, but will be much better for all of our customers in the long run.

Reply:
@ravbhatnagar Could you comment on ARM opinion on LROs?